docs/_data/menu.yml

```
@@ -66,10 +66,14 @@
       url: "/cluster-management/configure-queue-service"
     - title: "Configuring Build Cluster Queue Worker"
       url: "/cluster-management/configure-buildcluster-queue-worker"
+    - title: Configure the artifacts-unzip-service
```

it should be same with other pages.
```suggestion
    - title: Configuring the artifacts-unzip-service
```

---

Fixed